draft-packages/select/docs/Select.stories.tsx

```diff
@@ -55,6 +55,46 @@ DefaultSelectStory.args = {
   status: "default",
 }
 
+export const GroupedStory: ComponentStory<typeof Select> = args => {
+  const COLOUR_OPTIONS = [
+    { value: "blue", label: "blue" },
+    { value: "red", label: "red", isDisabled: true },
+    { value: "green", label: "green" },
+  ]
+
+  const FLAVOUR_OPTIONS = [
```

Comment:
🍨

Reply:
Is this you pointing out the spelling 🕵️